libexec/slurm/monitor.sh

```diff
@@ -73,11 +92,18 @@ elif [[ "$control" == "slurm_load_jobs error: Invalid job id specified" ]]; then
     reason=''
 
   # Extract the output from sacct
-  elif [[ "$exit_status" -eq 0 ]]; then
-    state=$(echo "$acct" | cut -d'|' -f1)
-    reason=$(echo "$acct" | cut -d'|' -f2)
+elif [[ "$exit_status" -eq 0 ]]; then
+  # sacct sometimes tacks info onto the end of the state
+  state=$(echo "$acct" | cut -d'|' -f1 | cut -d' ' -f1)
+  reason=$(echo "$acct" | cut -d'|' -f2)
+  if [ "$(echo "$acct" | cut -d'|' -f5)" ]; then
```

Comment:
It would be better to make the comparison to the empty string explicit here.  `if [ "$( ... )" != "" ]; then`.

Reply:
ping.

Reply:
The idiomatic way to do this in `bash` is to use the `-n` flag:

https://github.com/openflighthpc/flight-job/pull/14/commits/34c9a3cffe21b9d5dd7790319e979f5f9a271433

Reply:
`-n` is good too.  As long as we're not relying on the reader to recall that `[ "" ]` is false.